lib/logSearch.rb

```diff
@@ -0,0 +1,21 @@
+
+class LogSearch
```

Comment:
The class name is fine, but file names typically follow the convention 'log_search'.